packages/vike-react-chakra/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "vike-react-chakra",
+    "version": "0.0.1",
+    "type": "module",
+    "exports": {
+        "./config": "./dist/integration/+config.js",
+        "./__internal/integration/Wrapper": "./dist/integration/Wrapper.js"
+    },
+    "scripts": {
+        "dev": "tsc --watch",
+        "build": "rimraf dist/ && tsc",
+        "release": "release-me patch",
+        "release:minor": "release-me minor",
+        "release:commit": "release-me commit"
+    },
+    "peerDependencies": {
+        "@chakra-ui/react": ">=3",
+        "@emotion/react": ">=11",
+        "react": ">=18.0.0",
+        "vike-react": ">=0.4.13"
+    },
+    "devDependencies": {
+        "@brillout/release-me": "0.3.8",
+        "@chakra-ui/react": "^3.0.2",
+        "@emotion/react": "^11.13.3",
+        "@types/node": "^20.11.17",
+        "@types/react": "^18.2.55",
+        "jsdom": "^24.0.0",
+        "react": "^18.3.1",
+        "rimraf": "^5.0.5",
+        "typescript": "^5.5.3",
+        "vike": "^0.4.197",
+        "vike-react": "^0.5.7",
+        "vite": "^5.4.0"
+    },
+    "typesVersions": {
+        "*": {
+            "config": [
+                "dist/integration/+config.d.ts"
+            ],
+            "__internal/integration/Wrapper": [
+                "dist/integration/Wrapper.d.ts"
+            ]
+        }
+    },
+    "files": [
+        "dist"
+    ],
+    "repository": "github:vikejs/vike-react",
```

Comment:
Let's update this to be a more precise URL.

Reply:
AFAICT `"repository": "https://github.com/vikejs/vike-react/tree/main/packages/vike-react-chakra"` is enough.